crates/e2e/src/setup/colocation.rs

```diff
@@ -78,7 +78,7 @@ graph-api-base-url = "https://api.thegraph.com/subgraphs/name/"
 [[liquidity.uniswap-v2]]
 router = "{:?}"
 pool-code = "{:?}"
-missing-pool-cache-time-seconds = 3600
+missing-pool-cache-time = "10m"
```

Comment:
nit: The default changed from `1h` to `10m`.

Reply:
Overlooked, thanks